public/static/js/version.js

```diff
@@ -24,6 +24,11 @@ mciModule.controller('VersionController', function($scope, $rootScope, $location
       resource_type: "VERSION",
       label: "this version succeeds",
     },
+    {
```

Comment:
Don't expose this here.
1. it only make sense in the context of repotracker created versions.
2. Subscribe to test failures basically duplicates this functionality (aside from the relative lookup)

Reply:
removed